### DISCUSSION TITLE
"Configuring Log Retention in OpenTelemetry: Automatic Deletion of Logs in `config.yaml`"

### DISCUSSION THREAD

**Discussion by SandhyaSonkambleS** [open-telemetry/opentelemetry-collector-contrib#36434] [General]
I want to configure OpenTelemetry to automatically delete logs after a certain period. How can I define a rule in the config.yaml file to remove or manage logs based on their age? Is there a way to handle log retention directly within the configuration?

**Comment by adrien-dubois** [comment]
Hello, have you find a way? I'm also interested into
